cmd/soci/commands/internal/client.go

```diff
@@ -0,0 +1,68 @@
+/*
+   Copyright The Soci Snapshotter Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package internal
+
+import (
+	gocontext "context"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/containerd/containerd/pkg/epoch"
+	"github.com/urfave/cli"
+)
+
+// Largely taken from containerd/cmd/ctr/commands/client.go
```

Comment:
If this was copied, then this file needs the containerd copyright notice. I think this applies to all the files in `internal`, right?

For example, see https://github.com/awslabs/soci-snapshotter/blob/main/snapshot/snapshot.go#L1-L31.

Reply:
Sure. What about the files that use the same containerd flags?

Reply:
Yeah, I think they're word-for-word identical, aren't they? We should add the containerd copyright.